Tinylog/ViewModels/TasksViewModel.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  TasksViewModel.swift
+//  Tinylog
+//
+//  Created by Spiros Gerokostas on 19/7/20.
+//  Copyright © 2020 Spiros Gerokostas. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+final class TasksViewModel {
+    
+    private let managedObjectContext: NSManagedObjectContext
+    
+    init(managedObjectContext: NSManagedObjectContext) {
+        self.managedObjectContext = managedObjectContext
+    }
+    
+    private func getListTasks(with list: TLIList?) -> [TLITask]? {
+        guard let list = list else {
+            return nil
+        }
+        
+        do {
+            let fetchRequest: NSFetchRequest<TLITask> = NSFetchRequest(entityName: "Task")
+            let positionDescriptor: NSSortDescriptor = NSSortDescriptor(key: "position", ascending: false)
+            let displayLongTextDescriptor: NSSortDescriptor = NSSortDescriptor(key: "displayLongText",
+                                                                               ascending: true)
+            fetchRequest.sortDescriptors = [positionDescriptor, displayLongTextDescriptor]
+            fetchRequest.predicate = NSPredicate(format: "list = %@", list)
+            fetchRequest.fetchBatchSize = 20
+            return try managedObjectContext.fetch(fetchRequest) as [TLITask]
+        } catch let error as NSError {
+            fatalError(error.localizedDescription)
+        }
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/c61f36452fa17d076822a3b73830cc49c60091be/Tinylog/ViewModels/TasksViewModel.swift#L38"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TasksViewModel.swift:38</code></td>
    </tr>
  </tbody>
</table>